index.html

```diff
@@ -4,7 +4,111 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Personal Website</title>
+
+    <link rel="stylesheet" href="assets/styles/main.css" />
+    <link rel="stylesheet" href="assets/styles/header.css" />
+    <link rel="stylesheet" href="assets/styles/footer.css" />
+    <script
+      src="https://kit.fontawesome.com/0069c24e64.js"
+      crossorigin="anonymous"
+    ></script>
+
+    <title>Personal Website Christian Daum</title>
   </head>
-  <body></body>
+  <body>
+    <header>
+      <a class="navbar-brand" href=""> Christian Daum | WebDev</a>
+      <button class="navbar-toggler">
+        <span class="toggler-icon"></span>
+      </button>
+
+      <nav class="navbar">
+        <ul>
+          <li>
+            <a href="">Skills</a>
+          </li>
+          <li>
+            <a href="">Interests</a>
+          </li>
+          <li>
+            <a href="">Contact</a>
+          </li>
+          <li>
+            <a href="">Bio</a>
+          </li>
+          <li>
+            <a href="">Travels</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <!-- <section class="name"></section>
+    <section class="photo"></section>
+    <section class="about"></section>
+    <section class="work"></section>
+    <section class="education"></section>
+    <section class="community"></section>
+    <section class="skills"></section> -->
+
+    <section class="hero">
+      <h1>Hero</h1>
+      <h3></h3>
+    </section>
+
+    <section class="photo">
+      <h1>Pic</h1>
+      <h3></h3>
```

Comment:
Because you did this often I will not showing you all the sections. Please take a look and find the others headlines by yourself. :-)